src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -248,6 +247,144 @@ public static void FaultException_Throws_WithFaultDetail()
         Assert.True(false, "Expected FaultException<FaultDetail> exception, but no exception thrown.");
     }
 
+    [Theory]
```

Comment:
Can this be [Fact] instead of [Theory]?  It doesn't use any theory data to drive the test.  More importantly it won't be executed in ToF.